app/models/concerns/user_capabilities.rb

```diff
@@ -15,6 +15,6 @@ def user_capabilities
   end
 
   def user_context
-    Thread.current[:user]
+    UserContext::current_user_context
```

Comment:
Just for "ruby style" sake I think this should just be `UserContext.current_user_context`, otherwise this looks like it's trying to call a constant but named incorrectly.

Reply:
@eclarizio Nice catch that was a mistake, I have fixed it.